README.md

```diff
@@ -48,59 +49,168 @@ For .NET Core CLI, use the following command:
 
-#### Environments description
+### Check Platform Operational Status
+
+This endpoint offers a quick way to check the overall operational status of the platform.
+
+```csharp
+// Authentication
+var alfred = await AlfredFactory.CreateWithOAuthAsync("admin@tagshelf.com", "T@gSh3lf", EnvironmentType.Staging);
+
+// Get the Platform Operational Status
+TagshelfStatusResult status = await alfred.Tagshelf.StatusAsync();
+
+// Print the Result
+Console.WriteLine(status);
+```
+**Result**
+```cmd
+ApiVersion: 1.47.6.0, DbStatus: online
+```
+
+### Retrieve User Identity Information
+
+This endpoint provides information about the currently authenticated user.
+
+```csharp
+// Authentication
+var alfred = await AlfredFactory.CreateWithOAuthAsync("admin@tagshelf.com", "T@gSh3lf", EnvironmentType.Staging);
+
+// Get the Platform Operational Status
+WhoAmIResult whoami = await alfred.Tagshelf.WhoAmIAsync();
+
+// Print the Result
+Console.WriteLine(whoami);
+```
+**Result**
+```cmd
+UserName: admin@tagshelf.com, Name: Admin Tagshelf, Roles: Admin, Developer, Status: active, Company: TAGSHELF, App: , AppId: 00000000-0000-0000-0000-000000000000, RequestOriginIpAddress: 152.0.135.113:20845
+```
+
+### Conduct a Basic Connectivity Test
+
+The `ping` endpoint is used to perform a basic connectivity test to the platform.
+
+```csharp
+// Authentication
+var alfred = await AlfredFactory.CreateWithOAuthAsync("admin@tagshelf.com", "T@gSh3lf", EnvironmentType.Staging);
+
+// Get the Platform Operational Status
+PingResult ping = await alfred.Tagshelf.PingAsync();
+
+// Print the Result
+Console.WriteLine(ping);
+```
+**Result**
+```cmd
+Pong: admin@tagshelf.com
+```
 
-- **Production**: The production environment are used for live applications and real-world scenarios. It is recommended to use this environment for production-ready applications. The frontend URL for the production environment is [https://app.tagshelf.com](https://app.tagshelf.com)</br></br>
+## Deferred Sessions {#deferred-sessions-domain}
 
-- **Staging**: The staging environment is used for testing and development purposes. It is recommended to use this environment for testing and development scenarios. The frontend URL for the staging environment is [https://staging.tagshelf.com](https://staging.tagshelf.com)</br></br>
+A Deferred Session in Alfred is a mechanism designed for asynchronous file uploads, integral to the document processing workflow. It serves as a container or grouping for files that are uploaded at different times or from various sources, but are all part of a single processing task or Job.
 
-#### Authentication Methods
+### Create a new Deferred Session
```

Comment:
This title is empty. If we are not going to include information we should remove it entirely,